packages/cli/templates/content-block-tailwind/README.md

```diff
@@ -1 +1,43 @@
-# Frontify Block Boilerplate
+# Hello World
+This is a simple example of a Hello World app using React and Tailwind.
```

Comment:
I think this one is in the wrong folder, as its descibes a folder but this is a block

Reply:
haha indeed, sorry

Reply:
np all good now :)